sale_order_import/readme/CREDITS.rst

```diff
@@ -0,0 +1,3 @@
+The development of this module has been financially supported by:
```

Comment:
this is not required :wink:

Reply:
Hi, fyi, updated as `The migration of this module from 13.0 to 14.0 was financially supported by Camptocamp`